near-sdk-macros/src/lib.rs

```diff
@@ -137,3 +137,25 @@ pub fn metadata(item: TokenStream) -> TokenStream {
         )
     }
 }
+
+#[proc_macro_derive(NoDefault)]
```

Comment:
Please add docstring to explain its purpose in depth, so that docs.rs have nice documentation.